chaincfg/init_test.go

```diff
@@ -121,21 +121,21 @@ var (
 				Id:          "Abstain",
 				Description: "Abstain voting for Pedro",
 				Bits:        0x0,   // 0b0000
-				IsIgnore:    false, // XXX this is the invalid bit
+				IsAbstain:   false, // XXX this is the invalid bit
```

Comment:
The choice has the `Bits` field set to 0 however the `IsAbstain` flag set to false.  That is an invalid combination since a choice that doesn't have an bits set must be abstain.